src/main/java/com/turkcell/staj/entities/Package.java

```diff
@@ -0,0 +1,30 @@
+package com.turkcell.staj.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "packages")
+public class Package {
```

Comment:
Entity sınıf ismi 'package' java tarafından kullanıldığı için, sınıf ismi packages'a dönüştürülebilir

Reply:
"Packages" ismi de çoğul olduğu ve anlam karışıklıklarına yol açabileceği için "Offer" olarak değiştirmek daha iyi olabilir.

Reply:
Olur, çok iyi olur.

Reply:
Ben DTO'larda gerekli kısımları package'dan offer'a çekiyorum o zaman.